utilities/tools/AzureDevOps/Set-DevOpsPipelineFiles.ps1

```diff
@@ -0,0 +1,179 @@
+﻿
+<#
+.SYNOPSIS
+Generate all module pipeline files for Azure DevOps based on the ones currently implement for GitHub workflows
+
+.DESCRIPTION
+Generate all module pipeline files for Azure DevOps based on the ones currently implement for GitHub workflows
+Note: The parameter file refernces are fetched from the actual module folder. Hence the list may diviate from what is specified in the corrsponding GitHub workflow.
+
+.PARAMETER devOpsPipelineFolderPath
+Optional. The path where the Azure DevOps pipelines should be stored
+Defaults to: '.azuredevops/modulePipelines'
+
+.PARAMETER devOpsPipelineTemplatePath
+Optional. The path to the Azure DevOps pipeline template file.
+Defaults to: './devOpsPipelineTemplate.yml'
+
+Should contain the tokens (<...>) to be replaced.
+Currently supported are:
+- pipelineName
+- removeFlag
+- versionFlag
+- pipelineFileName
+- relativeModuleFolderPath
+- pipelineParamterPaths (should be set behind the 'deploymentBlocks: <...>')
+- relativePathOfRemovalScript (should bet set right under the removal template)
+
+.PARAMETER GitHubWorkflowFolderPath
+Optional. The path to the GitHub workflows folder to crawl from
+Defaults to: '.github/workflows'
+
+.EXAMPLE
+Set-DevOpsPipelineFiles
+
+Generate all Azure DevOps pipeline files in the default DevOps pipeline folder based on the workflows files in the default workflows folder based on the provided default template
+#>
+function Set-DevOpsPipelineFiles {
+
+
+    [CmdletBinding(SupportsShouldProcess)]
+    param (
+        [Parameter(Mandatory = $false)]
+        [string] $devOpsPipelineFolderPath = (Join-Path (Split-Path (Split-Path (Split-Path $PSScriptRoot))) '.azuredevops' 'modulePipelines'),
+
+        [Parameter(Mandatory = $false)]
+        [string] $devOpsPipelineTemplatePath = (Join-Path $PSScriptRoot 'src' 'devOpsPipelineTemplate.yml'),
+
+        [Parameter(Mandatory = $false)]
+        [string] $GitHubWorkflowFolderPath = (Join-Path (Split-Path (Split-Path (Split-Path $PSScriptRoot))) '.github' 'workflows')
```

Comment:
Pascal case parameters on a function.

[Parameter Design Guidelines (SD03) - Use Pascal Case for Parameter Names](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/strongly-encouraged-development-guidelines?view=powershell-7.2#use-pascal-case-for-parameter-names)

Reply:
done